src/PrestaShopBundle/Form/Admin/Sell/Address/CustomerAddressType.php

```diff
@@ -54,6 +56,8 @@
  */
 class CustomerAddressType extends AbstractType
 {
+    private const COMMON_INVALID_CHARS = ' <>;=#{}';
```

Comment:
```suggestion
    private const COMMON_INVALID_CHARS = '<>;=#{}';
```

Reply:
These are the tricky ones. Tested how the Customers / Addresses form validation works and noticed that not only ' ' (space char) is not validated but also ';' and '=' are not validated. Specification of how it should be would be very helpful.

Reply:
The pattern is TypedRegex::TYPE_GENERIC_NAME which has regex: 
`TypedRegex::TYPE_GENERIC_NAME => $this->characterCleaner->cleanNonUnicodeSupport('/^[^<>={}]*$/u'),`

Reply:
And in legacy iat was also without these chars: ' ;#':

>     public static function isGenericName($name)
>     {
>         return empty($name) || preg_match(Tools::cleanNonUnicodeSupport('/^[^<>={}]*$/u'), $name);
>     }

Reply:
```php
    $this->translator->trans('Invalid characters: %s', [self::COMMON_INVALID_CHARS], 'Admin.Notifications.Info'),
```
Better doing something like that please.
You maybe need to improve the `GENERIC_NAME` regex, or change this const.

Reply:
Created the constants inside TypedRegexValidator so that could reuse and not to duplicate the invalid chars string.